lib/queuesender/logger.go

```diff
@@ -0,0 +1,20 @@
+package queuesender
```

Comment:
You could use this instead: https://github.com/Symantec/Dominator/tree/master/lib/log/prefixlogger